### DISCUSSION TITLE
How to handle Stock and Bond Purchase

### DISCUSSION THREAD

**Discussion by fedex03** [firefly-iii/firefly-iii#7616] [Firefly III]
HI there,

I'm looking for the best apporch to handle with Stock and Bond purchase.
Shoud I convert the buy transaction into a transfer to a saving account? 

Thank you for the help,
Federico

**Comment by JC5** [comment]
Use another app, like PortfolioPerformance

**Reply by fedex03** [reply]
Hi @JC5 ,

yes sure, I'm using Ghostfolio.
I just want to know hwo to handle this scenario:

> I bought €1000 in stock and I don't want that these €1000 wil be counted as expenses because they are not.
> Looking at the summary for the month I would find higher expenses that confusing me.
> They are still "money" moved to another wallet.

Would it be correct to convert the "Stock Buy" expense into a transaction to a "Portfolio" wallet.
In this way my month expenses will be equal to my real expenses.

Thank you very much for your great work!

**Reply by JC5** [reply]
Ah yes, I use another asset account for that. I transfer the money there and then it's not an expense, but an "internal transfer".

**Reply by binick** [reply]
I am taking the same approach and am doing well.
However, I feel the lack of having reports that, like for transactions, can also break down assets by category or tags.
Is that in your plans?
